dashboard.el

```diff
@@ -41,13 +41,16 @@
     (define-key map (kbd "C-i") 'widget-forward)
     (define-key map [backtab] 'widget-backward)
     (define-key map (kbd "RET") 'dashboard-return)
-    (define-key map [down-mouse-1] 'widget-button-click)
     (define-key map (kbd "g") #'dashboard-refresh-buffer)
     (define-key map (kbd "}") #'dashboard-next-section)
     (define-key map (kbd "{") #'dashboard-previous-section)
     map)
   "Keymap for dashboard mode.")
 
+(if (display-graphic-p)
+    (define-key dashboard-mode-map [down-mouse-1] 'widget-button-click)
+  (define-key dashboard-mode-map [down-mouse-1] nil))
```

Comment:
Can you put this in the line of `defvar`? Maybe like

```el
(define-key map [down-mouse-1] (if (display-graphic-p) .. ..))
```

Reply:
After talking with some folks on freenode#emacs, it seems that, while possible to put a similar expression within the defvar expression, it would still require code outside the defvar expression.

Alternatively, could leave the defvar expression alone and follow it with this:
```lisp
(unless (display-graphic-p)
    (define-key dashboard-mode-map [down-mouse-1] nil))
```

Reply:
I don't think we need this part anymore since we removed the line L44. The default value should just be nil.

Reply:
yeah, i think so. i'll close this